docs/content/preview/releases/versioning.md

```diff
@@ -51,3 +51,75 @@ Patch releases in the preview release series (`MAJOR.ODD.PATCH`) introduce new f
 - There is currently no migration path from a preview release to a stable release.
 
 {{< /note >}}
+
+## Feature availability
+
+YugabyteDB features are made available as one of the following:
+
+- Tech Preview (TP)
+- Early Access (EA)
+- General Availability (GA)
+
+Changes for EA and GA are always reported in the [YugabyteDB Release Notes](../release-notes/).
```

Comment:
```suggestion
Changes for EA and GA are always reported in the [YugabyteDB Release Notes](../releases/).
```

Reply:
thats a broken link and if at all we report changes , I think it would be the releases homepage.